.github/workflows/release.yml

```diff
@@ -64,6 +64,11 @@ jobs:
       - name: Install dependencies
         uses: kubewarden/github-actions/policy-gh-action-dependencies@17f4dbf90d74f26892f7cc18179d08c4d63914d9 # v4.5.4
 
+      - name: Install javy
+        run: |
+          curl -L https://github.com/bytecodealliance/javy/releases/download/v5.0.4/javy-x86_64-linux-v5.0.4.gz -o javy.gz
+          gunzip javy.gz && chmod +x javy && sudo mv javy /usr/local/bin/
+
```

Comment:
Please, use the GITHUB_PATh to install the binary in the system path. It's not necessary to change the system directories. 

You can find an example of this [here](https://github.com/kubewarden/github-actions/blob/20e6274f09e9aee263969a8783488369cbdb1e1b/crane-installer/action.yml#L16-L27)